origami/src/main/java/origami/Epsilon.java

```diff
@@ -30,13 +30,12 @@ public class Epsilon {
     public static final double UNKNOWN_1EN5 = factor * 1E-5;
     public static final double UNKNOWN_1EN6 = factor * 1E-6;
     public static final double UNKNOWN_1EN7 = factor * 1E-7;
-
     // These are the constants with a known purpose.
 
     public static final double PARALLEL = factor * 0.5;
     public static final double FLAT = factor * 1E-4;
     public static final double QUAD_TREE_ITEM = factor * 0.5;
-
+    public static final double GRID_ANGLE_THRESHOLD = 5;
```

Comment:
I think 5 might be a bit too much, maybe 1 or even 0.5 would be better.

Reply:
Updated the threshold to 1.

Reply:
Smaller angles also cause performance issues on slow machines; e.g mine hangs at 1 degree; and the lines look almost much parallel at 10x zoom